firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+FindInPage.swift

```diff
@@ -21,6 +21,7 @@ extension BrowserViewController {
         guard let webView = tabManager.selectedTab?.webView else { return }
 
         if isVisible {
+            webView.scrollView.keyboardDismissMode = .none
```

Comment:
Shouldn't this be reset to the original `keyboardDismissMode` after Find in page is dismissed?